security.sh

```diff
@@ -3,11 +3,13 @@
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
 export PYTHONDONTWRITEBYTECODE=1
-
 zap-api-scan.py -t ${URL_FOR_SECURITY_SCAN}/v2/api-docs -f openapi -S -d -u ${SECURITY_RULES} -P 1001 -l FAIL --hook=zap_hooks.py
-cat zap.out
 curl --fail http://0.0.0.0:1001/OTHER/core/other/jsonreport/?formMethod=GET --output report.json
+
+export LC_ALL=C.UTF-8
```

Comment:
same on lines 3-5

Reply:
I was led to believe that the script needed the duplication due to a recent ZAP docker update, and my testing seemed to bear that out too - but on re-trying this morning I'm getting different results.
I'd seen some other Jenkins issues late yesterday, so that might've skewed my results, as I can now see the script working as expected.
I've changed this to just log those env vars for future debugging